tools/xmc-flasher.py

```diff
@@ -10,6 +10,10 @@
 cmd_jlink =  os.path.join(tempfile.gettempdir(), 'cmd.jlink')
 console_out = os.path.join(tempfile.gettempdir(), 'console.output')
 
+# Suppress traceback for Arduino IDE
+# !! comment out if you want to see the traceback for debug/ developement !!
+sys.tracebacklimit = 0
```

Comment:
Rather than suppressing the traceback we should have a solid error handling, or? After the added exception handling, we still get traceback errors?

Reply:
Alternatively, this could be made a cli option flag ? Enabled by arduino-cli flags or IDE debug flags?

Reply:
cli flag could be a good idea

Reply:
add a --verbose flag for xmcflasher. This also applies to Arduino IDE and arduino-cli (tested on windows).
Once merged into bugfix/issue #266, I can test again on all operating systems